src/lib/abacus/dydxChainTransactions.ts

```diff
@@ -238,9 +244,10 @@ class DydxChainTransactions implements AbacusDYDXChainTransactionsProtocol {
 
       if (isTestnet) {
         console.log(
-          `${ENVIRONMENT_CONFIG_MAP[
-            this.compositeClient.network.getString() as DydxNetwork
-          ]?.links?.mintscan?.replace('{tx_hash}', hash.toString())}`
+          `${LINKS_CONFIG_MAP[
+            ENVIRONMENT_CONFIG_MAP[this.compositeClient.network.getString() as DydxNetwork]
+              .dydxChainId as DydxChainId
+          ]?.mintscan?.replace('{tx_hash}', hash.toString())}`
```

Comment:
This works but maybe we can add the link as a `get` var, outside of this method.